Echo.Process/Prelude_Session.cs

```diff
@@ -300,8 +300,7 @@ public static R withSession<R>(SessionId sid, Func<R> f)
         /// <returns></returns>
         public static Unit addSupplementarySession(SupplementarySessionId sid) =>
             hasActiveSession()
-                ? sessionGetData<SupplementarySessionId>(SupplementarySessionId.Key)
-                    .HeadOrNone()
+                ? Optional(sessionGetData<SupplementarySessionId>(SupplementarySessionId.Key).LastOrDefault())
```

Comment:
The SessionVector, incase of conflict tries to add data to end of the list. There is a rare chance of that conflict happening as per current use cases, but it can happen, so that's why I had to change to ugly LastOrDefault..

Reply:
Using Seq.LastOrNone now